test/README.md

```diff
@@ -17,13 +17,29 @@ a list of cells to ignore for each notebook, which is useful when specific cells
 intentionally erroneous. Each notebook is then loaded in a browser page, and all its cells are
 executed, and tests pass if no errors are produced.
 
-Tests were written as a nodejs package, so in order to run them locally, you can do `npm install`
-then `npm test` (or `npm install-test`). The one dependency is you have to be able to install
+## Running Tests
+
+### TL;DR
+
+Make sure you can install cairo on your system. Check the `.travis.yml` file for more info
+`cd test`
+`npm install-test`
+Or
+`npm test` if you have already installed the dependencies once
```

Comment:
Get rid of the comment here. If you have separate sections for one-time setup and for running tests, it will be clear that one-time setup has to be done before running tests.